docs/guides/no-low-code-experiments.md

```diff
@@ -0,0 +1,101 @@
+# Running no and low code experiments in Eppo
+
+Often times there are frequently tested parts of a site or app where teams would like to execute changes in a self-service manner. Running experiments without an additional code is easy to do in Eppo with some initial set up. With a JSON flag and parameterizing parts of the site or app where frequent changes are made, users can create additional experiment allocations on the same flag/
+
+## Creating multiple experiments on the same flag
+
+For this example let’s use the following feature flag in Eppo. There are 2 allocations; one is a feature gate that prioritizes sending any internal users to the treatment variation, and the other is an experiment allocation that evaluates whether the other users meet the audience criteria. If users meet the experiment audience criteria, they will be split 50-50 between the control and treatment variations.
+
+![Initial experiment allocation on a flag](/img/guides/no-low-code-experiments/first-exp-allocation.png)
+
+To demonstrate how to run multiple experiments on the same flag, create a new variation by editing your feature flag and clicking **Add Variation**.
+
+![Add a variation to a flag](/img/guides/no-low-code-experiments/add-variation.png)
+
+Once the new variation is added, go back to the feature flag overview page and create a new experiment allocation by clicking **Add Allocation** and selecting **Experiment**.
+
+After adding an additional Experiment Allocation, the allocation shows up on your Feature Flag overview page.
+
+![Second experiment allocation on a flag](/img/guides/no-low-code-experiments/second-exp-allocation.png)
+
+
+This new allocation logic splits the remaining users 50/50 into the second and third variations if those users do not qualify for the first 2 allocations. The previous experiment allocation can be archived by selecting the ‘bin’ icon on the right hand side of the allocation. Additionally, by clicking the gray side bar on the right hand side of the allocation, allocations can be dragged and dropped to reprioritize the order of logic executed.
+
+:::warning
+
+Eppo does not recommend running two experiments on the same flag at the same time since it can interfere with experiment results. We recommend archiving the experiment allocation after the test is complete before adding a new experiment allocation to the same flag. The example above is to illustrate the ability to have multiple experiments created on the same flag.
```

Comment:
This seems odd to me - the guide is showing how to do it, but then we say we actually don't recommend doing it.

Is the main concern that a user could drift from one experiment to another? Maybe this warning should really say to only run this kind of experiment when users are unlikely to change populations (e.g., if the targeting is based on primary country, etc.)

Reply:
Yea, i went back and forth on this--maybe I just make the stated workflow "archive the previous experiment, and now you're running a new experiment on the same flag" and remove the warning.

Last time we went through this, there was a strong opinion to make sure users were aware of the downstream effects of running multiple concurrent experiments on the same flag.

Reply:
ended up taking out for now